src/app/(dashboard)/profile/page.tsx

```diff
@@ -72,12 +74,14 @@ export default function ProfileScreen() {
     <div className="container mx-auto p-6">
       <div className="mb-6 flex items-center space-x-4">
         <Avatar className="h-20 w-20">
-          <AvatarImage src="/avatars/01.png" alt="Profile picture" />
+          <AvatarImage src={user?.imageUrl} alt="Profile picture" />
           <AvatarFallback>JD</AvatarFallback>
```

Comment:
change fallback to user's first name and last name  initials , i would recommend to check clerk doc if they have any utility function if not please make one in utilities folder